packages/modular-scripts/src/check.ts

```diff
@@ -111,14 +114,26 @@ export async function check(): Promise<void> {
     }
   });
 
-  /**
-   * Taken from react-scripts - this assets that we share common versions of utils which we depend on.
-   */
   const { verifyPackageTree } = await import('./utils/verifyPackageTree');
+  failed = await verifyPackageTree();
+
   try {
-    await verifyPackageTree();
+    require.resolve('typescript');
```

Comment:
This seems like it should belong in verifyPackageTree? If typescript is required, why not put it in `DEPS_TO_CHECK`?

Reply:
Or as a dependency?

Reply:
We don't enforce which version of `typescript` you have to use - since it would tie upgrade of `typescript` to `modular-scripts` itself, but rather need to ensure that it's present for command to work properly.